packages/react/src/components/Storage/FileUploader/UploaderButton/index.tsx

```diff
@@ -1,22 +1,38 @@
-/* eslint-disable @typescript-eslint/no-unused-vars */
 import React from 'react';
-import { Button } from '../../../../primitives/Button';
+import { translate } from '@aws-amplify/ui';
 import { UploaderButtonProps } from '../types';
+import { Button, VisuallyHidden } from '../../../../primitives';
 
 export function UploaderButton({
   multiple,
   acceptedFileTypes,
-  onClick,
+  onFileChange,
```

Comment:
What does `onFileChange` mean? Think we can just pass a click event handler here and move the logic inside the component in to its parent

Reply:
The `onFileChange` is not a click handler, it's a change event that is on the input that's passed in from the parent.